README.md

```diff
@@ -103,6 +103,12 @@ Some of the options accept parameters:
 exa is available for macOS and Linux.
 More information on how to install exa is available on [the Installation page](https://the.exa.website/install).
 
+### Alpine Linux
+
+On Alpine Linux, install the [`exa`](https://pkgs.alpinelinux.org/package/edge/community/x86_64/exa) package.
```

Comment:
I think it’s better to mention that it’s in the community repository, and just link to Alpine’s wiki:

```suggestion
On Alpine Linux, [enable community repository](https://wiki.alpinelinux.org/wiki/Enable_Community_Repository) and install the [`exa`](https://pkgs.alpinelinux.org/package/edge/community/x86_64/exa) package.
```